mlprimitives/jsons/sklearn.linear_model.Lasso.json

```diff
@@ -0,0 +1,129 @@
+{
+    "name": "sklearn.linear_model.Lasso",
+    "contributors": ["Katherine Wang <wang19k@mit.edu>"],
+    "documentation": "https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Lasso.html#sklearn.linear_model.Lasso",
+    "description": "Linear Model trained with L1 prior as regularizer (aka the Lasso)",
+    "classifiers": {
+        "type": "estimator",
+        "subtype": "regressor"
+    },
+    "modalities": [],
+    "primitive": "sklearn.linear_model.Lasso",
+    "fit": {
+        "method": "fit",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            },
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "produce": {
+        "method": "predict",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "hyperparameters": {
+        "fixed": {
+            "copy_X": {
+                "type": "bool",
+                "description": "If True, X will be copied; else, it may be overwritten.",
+                "default": true
+            },
+            "max_iter": {
+                "type": "int",
+                "description": "The maximum number of iterations",
+                "default": 1000
+            },
+            "warm_start": {
+                "type": "bool",
+                "description": "When set to True, reuse solution of the previous call to fit as initialization, otherwise, erase the previous solution.",
+                "default": false
+            },
+            "random_state": {
+                "type": "multitype",
+                "description": "The seed of the pseudo random number generator to use when shuffling the data.",
+                "types": {
+                    "int":{
+                        "description": "random_state is the seed used by the random number generator"
+                    },
+                    "RandomState":{
+                        "description": "If an array is passed, penalties are assumed to be specific to the targets. Hence they must correspond in number."
+                    },
+                    "NoneType": {
+                        "description": "If none, the random number generator is the RandomState instance used by np.random"
+                    }
+                },
+                "default": null
+            },
+            "precompute": {
+                "type": "multitype",
+                "description": "Whether to use a precomputed Gram matrix to speed up calculations. Gram natrix can also be passed as argument.",
```

Comment:
The scikit-learn documentation is not precise enough about it, but it seems to me that `"auto"` is also a valid value. Can you add it?